plugins/pulp_rpm/plugins/migrations/0016_new_yum_distributor.py

```diff
@@ -176,3 +177,58 @@ def _re_publish_repository(repo, distributor):
 def _remove_legacy_publish_dirs():
     _clear_orphaned_publish_dirs(OLD_ROOT_PUBLISH_DIR, OLD_HTTP_PUBLISH_DIR)
     _clear_orphaned_publish_dirs(OLD_ROOT_PUBLISH_DIR, OLD_HTTPS_PUBLISH_DIR)
+
+
+def _working_dir_root():
+    storage_dir = pulp_config.config.get('server', 'storage_dir')
+    dir_root = os.path.join(storage_dir, 'working')
+    return dir_root
```

Comment:
Why not eliminate this function and move its code into repository_working_dir() below?